versions.sh

```diff
@@ -2,7 +2,7 @@
 
 # See this for current versions: https://www.nvidia.com/object/unix.html
 BETA_VERSIONS="455.23.04 450.51 440.26 435.17 430.09 418.30 415.13 410.57 396.18"
-VULKAN_VERSIONS="455.26.02 455.26.01 455.22.04 450.56.11 450.56.06 450.56.02 450.56.01 440.66.17 440.66.15 440.66.14 440.66.12 440.66.11 440.66.09 440.66.08 440.66.07 440.66.04 440.66.03 440.66.02 440.58.02 440.58.01 440.48.02 440.43.02 440.43.01 435.27.08 435.27.07 435.27.06 435.27.03 435.27.02 435.27.01 435.27.00 435.24.02 435.19.03 435.19.02 418.52.20 418.52.18 418.52.17 418.52.16 418.52.14 418.52.10 418.52.07 418.52.05 418.52.03 418.49.04 418.42.02 418.31.03 415.22.05 415.22.01 415.18.04 415.18.02"
+VULKAN_VERSIONS="455.26.02 455.26.01 455.22.04 450.56.11 450.56.06 450.56.02 450.56.01 440.66.17 440.66.15 440.66.14 440.66.12 440.66.11 440.66.09 440.66.08 440.66.07 440.66.04 440.66.03 440.66.02 440.58.02 440.58.01 440.48.02 440.43.02 440.43.01 435.27.08 435.27.07 435.27.06 435.27.03 435.27.02 435.27.01 435.27.00 435.24.02 435.19.03 435.19.02 418.52.20 418.52.18 418.52.17 418.52.16 418.52.14 418.52.10 418.52.07 418.52.05 418.52.03 418.49.04 418.42.02 418.31.03 415.22.05 415.22.01 415.18.04 415.18.02 455.34.01"
```

Comment:
Would you move the new version to the beginning of the list?

Reply:
Right. Didn't notice reverse order.